File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/presentation/ui/base/activity/BaseSearchPlaceActivity.java
package com.lattis.ellipse.presentation.ui.base.activity;

import android.os.Bundle;
import androidx.annotation.Nullable;

import com.google.android.gms.common.api.GoogleApiClient;

/**
 * Created by lattis on 29/08/17.
 */

public abstract class BaseSearchPlaceActivity < Presenter extends ActivityPresenter> extends BaseCloseActivity
implements GoogleApiClient.OnConnectionFailedListener{
    GoogleApiClient mGoogleApiClient;



}

File: velolabs_repos/lattis_android_v2/app/src/main/java/com/lattis/lattis/presentation/base/activity/BaseSearchPlaceActivity.kt
package com.lattis.lattis.presentation.base.activity

import com.google.android.gms.common.api.GoogleApiClient
import com.google.android.gms.common.api.GoogleApiClient.OnConnectionFailedListener
import com.lattis.lattis.presentation.base.BaseView

abstract class BaseSearchPlaceActivity<Presenter : ActivityPresenter<V>,V:BaseView> :
    BaseCloseActivity<Presenter,V>(), OnConnectionFailedListener {
    var mGoogleApiClient: GoogleApiClient? = null
}